MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -581,10 +589,37 @@ - (MUXSDKPlayerData *)getPlayerData {
                                                                     encoding:NSUTF8StringEncoding]];
         }
     }
+
+    // Only report sampling data if the playback is live
```

Comment:
let's update this comment to say `Only report program time metrics if playback is live` (since we no longer have a sampling event)

Reply:
Actually, a more accurate comment would be `Only report program time metrics if this is a live stream`